app/services/idv/usps_mail.rb

```diff
@@ -0,0 +1,36 @@
+module Idv
+  class UspsMail
+    def initialize(current_user)
+      @current_user = current_user
+      @max_events = 4
+    end
+
+    def mail_spammed?
+      any_mail_events? && correct_length && updated_within_last_month
+    end
+
+    private
+
+    def find_user_events(event_type, user = @current_user)
```

Comment:
lets remove `@current_user` as a default argument since this method is never called without an explicit user